frontend/document/credocument.lua

```diff
@@ -310,6 +310,7 @@ function CreDocument:close()
         self.buffer = nil
     end
 
+    self.been_rendered = false
```

Comment:
May be this would work the same, but keeping your changed limited to readerrolling:
```diff
 function ReaderRolling:onCloseDocument()
+    self.current_header_height = nil -- show unload progress bar at top

```